lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -2630,11 +2630,12 @@ GetFrameSetupInfo(UnwindPlan &unwind_plan, RegisterContext &regctx) {
     LLDB_LOG(GetLog(LLDBLog::Unwind), "SwiftLanguageRuntime:: UnwindPlan did "
                                       "not contain a valid row after FP setup");
     auto *row = unwind_plan.GetRowAtIndex(row_idx);
-    return FrameSetupInfo{row->GetOffset(), fp_locs[row_idx].GetOffset()};
+    return FrameSetupInfo{static_cast<addr_t>(row->GetOffset()),
+                          fp_locs[row_idx].GetOffset()};
   }
 
   auto *subsequent_row = unwind_plan.GetRowAtIndex(next_row_idx);
-  return FrameSetupInfo{subsequent_row->GetOffset(),
+  return FrameSetupInfo{static_cast<addr_t>(subsequent_row->GetOffset()),
                         fp_locs[next_row_idx].GetOffset()};
 }
```

Comment:
Re: 49952f9c818b126795832a877257b07c0a8f933b

I’ve checked this in for review, but I don’t think it’s the right action to take.